fbchat/client.py

```diff
@@ -989,6 +991,15 @@ def fetchPlanInfo(self, plan_id):
         plan = graphql_to_plan(j["payload"])
         return plan
 
+    def getUserActiveStatus(self, user_id):
```

Comment:
We need to include some information about that we can only get this information when the listener loop is running

Reply:
Should we raise an error if it isn't?